Makefile

```diff
@@ -18,89 +18,51 @@
 # bingo manages go binaries needed for building the project
 include .bingo/Variables.mk
 
-GETENVOY_TAG ?= dev
+##@ Binary distribution
 
-BUILD_DIR ?= build
-BIN_DIR ?= $(BUILD_DIR)/bin
+.PHONY: release
+release:
+	@echo "--- release ---"
+	@$(GORELEASER) release --rm-dist
 
 GOOS := $(shell go env GOOS)
 GOARCH := $(shell go env GOARCH)
+BIN := dist/getenvoy_$(GOOS)_$(GOARCH)
+bin $(BIN): $(GORELEASER)
+	@echo "--- bin ---"
+	@$(GORELEASER) build --snapshot --rm-dist
 
-GO_LD_FLAGS := -ldflags="-s -w -X github.com/tetratelabs/getenvoy/pkg/version.version=$(GETENVOY_TAG)"
-
-TEST_PKG_LIST ?= $(shell go list ./... | grep -v github.com/tetratelabs/getenvoy/test/e2e)
-GO_TEST_OPTS ?=
-GO_TEST_EXTRA_OPTS ?=
-
-E2E_PKG_LIST ?= ./test/e2e
-# Run only one test at a time, in verbose mode, so that failures are easy to diagnose.
-# Note: -failfast helps as it stops at the first error. However, it is not a cacheable flag, so runs won't cache.
-E2E_OPTS ?= -parallel 1 -v -failfast
```

Comment:
in general I removed all the variables as we should focus on the makefile working by default (overrides unnecessary). Before, all the variables cluttered the file and made it easy to not see what actually was happening